optuna/integration/lightgbm.py

```diff
@@ -62,60 +64,83 @@ class LightGBMPruningCallback(object):
             Note that this argument will be ignored if you are calling
             `cv method <https://lightgbm.readthedocs.io/en/latest/Python-API.html#lightgbm.cv>`_
             instead of train method.
+        report_interval:
+            Check if the trial should report intermediate values for pruning every n-th boosting iteration.
```

Comment:
Seems like this line is too long. https://github.com/optuna/optuna/pull/2490/checks?check_run_id=3775102398